internal/proxy/hook/authz/authz.go

```diff
@@ -71,8 +64,13 @@ func (a Authz) ServeHook(res *http.Response, err error) (*http.Response, error)
 		return a.escape.ServeHook(res, err)
 	}
 
-	ruleFromRequest, ok := hook.ExtractRule(res.Request)
+	attributes, ok := attributes2.GetAttributesFromContext(res.Request.Context())
```

Comment:
can you do the change that we discussed for this naming? var can be renamed to `attribs` or `attribMap` or something like that.